lib/model.js

```diff
@@ -1079,7 +1078,15 @@ Model.exists = function exists(filter, options, callback) {
     });
     return;
   }
-
+  console.log(options);
+  if (typeof options !== 'undefined') {
+    if (!('explain' in options)) {
```

Comment:
I think this should be `if (!options.explain)`, because `explain` can be `false` as well.